File: dotnet/windntrees.net/Application/Scripts/ViewModels/LicenseInfo.js
function LicenseInfo(data) {
    var instance = this;

    instance._datakey = data.ProductID;
    instance.ProductID = data.ProductID;

    instance.Code = ko.observable(data.Code);
    instance.Code.extend({
        maxLength: 1024
    });

    instance.RowVersion = data.RowVersion;
}

File: dotnet/windntrees.core/DataAccess.Core/Models/LicenseInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DataAccess.Core.Models
{
    public partial class LicenseInfo
    {
        [Key]
        [Column("ProductID")]
        public Guid ProductId { get; set; }
        [Required]
        public string Code { get; set; }
        public byte[] RowVersion { get; set; }

        [ForeignKey("ProductId")]
        [InverseProperty("LicenseInfo")]
        public Product Product { get; set; }
    }
}
